src/plugins/intel_cpu/src/nodes/mkldnn_fullyconnected_node.cpp

```diff
@@ -330,6 +436,11 @@ std::shared_ptr<MemoryDesc> MKLDNNFullyConnectedNode::getSrcMemDesc(mkldnn::prim
         return std::make_shared<CpuBlockedMemoryDesc>(MKLDNNExtensionUtils::DataTypeToIEPrecision(
             static_cast<mkldnn::memory::data_type>(desc.data.data_type)), getInputShapeAtPort(idx));
     }
+
+    if (getInputShapeAtPort(idx).isDynamic()) {
+        return MKLDNNExtensionUtils::makeUndefinedDesc(desc, getInputShapeAtPort(idx));
+    }
```

Comment:
M.b. we need to set dims on the bias axis with the bias shape just like we do it in `getSupportedDescriptors()`

Reply:
Could you clarify please why we need this? For other oneDnn nodes we did it the same way.

Reply:
We did it without taking into account cases where we have undefined dim in activations tensor shape that must be aligned with one of the constant inputs. Remember fixes for the convolution node. So may be it makes sense to do the same trick also here?

Reply:
We already have such trick for FullyConnected. In `getSupportedDescriptors` we create defined descs: https://github.com/openvinotoolkit/openvino/blob/16b260bee589c31244bf0e61cfd9aa94dd870e12/src/plugins/intel_cpu/src/nodes/mkldnn_fullyconnected_node.cpp#L131-L145.
In `createDescriptor` we also create defined descs: https://github.com/openvinotoolkit/openvino/blob/16b260bee589c31244bf0e61cfd9aa94dd870e12/src/plugins/intel_cpu/src/nodes/mkldnn_fullyconnected_node.cpp#L422-L436